CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ let clocks = rcc
 
 - The feature gate requires you to select a subvariant if possible. ([#75](https://github.com/stm32-rs/stm32f3xx-hal/pull/75))
 - Split up `stm32f302` into sub-targets `stm32f302xb`,`stm32f302xc`,`stm32f302xd`,`stm32f302xe`
+- Remove the `stm32f301` sub-targets `stm32f301xb`,`stm32f301xc`,`stm32f301xd`,`stm32f301xe`
```

Comment:
These subtargets where introduced in https://github.com/stm32-rs/stm32f3xx-hal/pull/75. 
As we did not release any version since then, this change does not have to go into the changelog.